docs/sources/installation/helm/monitor-and-alert/index.md

```diff
@@ -80,6 +80,25 @@ prometheus:
         targetLabel: cluster
 ```
 
+In order to make sure the Prometheus Operator discovers the `ServiceMonitor` resources deployed by the `loki` chart, you will need to make sure those resources have the correct labels the Prometheus Operator is configured to look for. By default this is the key value label pair `release: prometheus`. Make sure the Loki `ServiceMonitor`s have this label by adding the following to the `values.yaml` for your Loki helm chart:
+
+```yaml
+monitoring:
+  serviceMonitor:
+    labels:
+      release: prometheus
```

Comment:
unfortunately the helm-chart I used to install the prom operator used a different labelset

```
release: prom-stack
```

Maybe we can include instructions on how to figure out what label is needed?

Reply:
@slim-bean which helm chart did you use? did you not use `kube-prometheus-stack`?

Reply:
I think we should keep the instructions on `kube-prometheus-stack` as that install `kube-state-metrics` with the correct recodring rules, `node-exporter`, and the Prometheus Operator. One stop shop for everything someone needs to enable self-monitoring.